### DISCUSSION TITLE
Tag Tabs

### DISCUSSION THREAD

**Discussion by helsengreen** [runelite/runelite#15380] [Q&A] [answered]
I made my Tagtabs on my ironman while doing NMZ on my main yesterday. When i logged on today it seems like it saved the config from my main and ive lost all my tag tabs. Is there anyway to get them back?

**Comment by Nightfirecat** [comment, marked as answer]
No. Running multiple clients at once means that your configs changed on one client can be overwritten by those on another client depending on the order they are closed in. We don't currently have a way to support multiple clients making simultaneous changes to config at once.
